src/ui/component/collapsible/collapsible.scss

```diff
@@ -29,17 +38,32 @@
 
 .okp4-dataverse-portal-collapsible-content {
   overflow: hidden;
+  
+  &.floating-up {
+    top: 0;
+    transform: translateY(-100%);
+  }
+  
+  &.floating-down {
+    top: 100%;
+  }
+
+  &.floating {
+    position: absolute;
+    z-index: 1;
+  }
+  
```

Comment:
to remain consistent with my remark about classnames, we could update this with:

```
  &.floating {
    position: absolute;
    z-index: 1;
    
    &.up {
      top: 0;
      transform: translateY(-100%);
    }
    
    &.down {
      top: 100%;
    }
  }
```